Components/rating/styles.css

```diff
@@ -0,0 +1,48 @@
+@import url(/rootStyles.css);
+
+.rating-container {
+  display: flex;
+  flex-direction: row-reverse;
+  width: fit-content;
+}
+
+
+.star {
+  font-size: 2rem;
+  color: var(--background-color);
+  appearance: none;
+}
+.r1,
```

Comment:
Use a single common class instead. Code is looking pretty messy because of that, as seen the same practice below on 'hover' and 'checked'.

Reply:
these are targetting individual elements, if i use a common class name we wont be able to do that